examples/trigger/trigger.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2018-2020 opcua authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/gopcua/opcua"
+	"github.com/gopcua/opcua/debug"
+	"github.com/gopcua/opcua/id"
+	"github.com/gopcua/opcua/ua"
+)
+
+func main() {
+	var (
+		endpoint      = flag.String("endpoint", "opc.tcp://morchard-mbp1.local:53530/OPCUA/SimulationServer", "OPC UA Endpoint URL")
```

Comment:
odds that my laptop is called `morchard-mbp1` is quite low :)

Reply:
lol, dang. ok, most of these look like they suffer from classic late Friday checkin syndrome. my apologies, I updated this to match the default in the other examples.